staging/src/k8s.io/component-base/cli/globalflag/globalflags.go

```diff
@@ -66,3 +68,20 @@ func Register(local *pflag.FlagSet, globalName string) {
 		panic(fmt.Sprintf("failed to find flag in global flagset (flag): %s", globalName))
 	}
 }
+
+// RegisterGlobalPflag adds a flag to local that targets the Value associated with the Flag
+// named globalName in pflag.CommandLine.
+func RegisterGlobalPflag(local *pflag.FlagSet, globalName string) {
```

Comment:
This name is a little redundant given that we're already in the `globalflags` package. Suggest `RegisterPflag` as an alternative.

Reply:
fix